libs/SalesforceAnalytics/SalesforceAnalytics/Classes/Logger/SFSDKLogger.h

```diff
@@ -144,4 +197,103 @@
  */
 - (void)log:(nonnull Class)class level:(DDLogLevel)level format:(nonnull NSString *)format, ...;
 
+/**
```

Comment:
I'd rather not have class-level conveniences, at least not for setting and getting log level. The behavior can be misleading to developers especially based on our 5.2 documentation. For example, a developer might use `[SFSDKSmartStoreLogger setLogLevel:...]` thinking that he/she is setting the log level for `SmartStore`, but it's actually being set for the default component. Same goes for any of the class level logging methods.